banzai/main.py

```diff
@@ -45,6 +49,10 @@
                   astrometry.WCSSolver,
```

Comment:
`ordered_stages` should probably be all caps too. Your favourite!

Reply:
You are of course correct.